csss-site/src/csss/templates/csss/new_header.html

```diff
@@ -97,6 +97,8 @@
             <a href="{{URL_ROOT}}events/board_games" class="navbar-item">What are Board Games Night?</a>            
             <a href='{{ URL_ROOT}}events/frosh/' class="navbar-item">CSSS FroshWeek</a>
             <a href="{{URL_ROOT}}events/mountain_madness2020" class="navbar-item">Mountain Madness 2020</a>          
+            <a href="{{URL_ROOT}}events/mountain_madness2021" class="navbar-item">Mountain Madness 2021</a>          
+            <a href="{{URL_ROOT}}events/fall_hacks2020" class="navbar-item">Fall Hacks 2020 Projects</a>          
```

Comment:
can you order these chronologically?

Reply:
No, the frosh events are not ordered chronologically, they should be grouped by event type/name then by date.